src/lib/marketingCollectionsCategories.ts

```diff
@@ -0,0 +1,70 @@
+export const marketingCollectionsByCategory = {
```

Comment:
**suggestion**: This file will likely only export a single data structure. Thoughts on renamed this const to match the filename and exporting a default?

For the filename, why the double plural? "Collection categories" reads a bit better to me than "collections categories". Similar to "artwork medium" or "attribution classes".

```ts
const marketingCollectionCategories = {}

export default marketingCollectionCategories
```

Reply:
This diff introduces a second hardcoded data structures for the same concept. Can we migrate the previously defined structure in the [Explore By section instance definition](https://github.com/artsy/metaphysics/blob/3b7d326deb1b11130b5a849c5722dd3a6ab8dc9b/src/schema/v2/homeView/sections/ExploreByCategory.ts#L6-L40) and consolidate here?

Reply:
We have a single concept "Collection Categories" and we're creating a new file here to define those categories (matching those used in Gravity).

We already have some of their attributes defined in [ExploreByCategory.ts](https://github.com/artsy/metaphysics/blob/3b7d326deb1b11130b5a849c5722dd3a6ab8dc9b/src/schema/v2/homeView/sections/ExploreByCategory.ts#L6-L40), like id, title, imageUrl. Here's we're specifying another attribute: an ordered/curated set of collection slugs.

I'd recommend that we combine all of this metadata into this new file. If we need to still specify a separate ordering for the ExploreByCategory section instance, we can do that by referencing category definition in the file.

For example:

```ts
import marketingColletionCategories from "lib/marketingCollectionCategories"

const categoryKeys = [
  "Medium",
  "Movement",
  "Collect by Size",
  "Collect by Color",
  "Collect by Price",
  "Gallery"
]

export const ExploreByCategory: HomeViewSection = {
  id: "home-view-section-explore-by-category",
  featureFlag: "diamond_home-view-marketing-collection-categories",
  contextModule: ContextModule.exploreBy,
  type: HomeViewSectionTypeNames.HomeViewSectionCards,
  component: {
    title: "Explore by Category",
  },
  requiresAuthentication: false,
  resolver: (_parent, args, _context, _info) => {
    const cards = categoryKeys.map((key) => {
      const category = marketingColletionCategories[key]

      return {
        entityID: category.id,
        entityType: OwnerType.collectionsCategory,
        image_url: category.imageUrl,
        title: category.title,
      }
    })

    return connectionFromArray(cards, args)
  },
}
```

Reply:
Thanks for clarifying that makes more sense!